modules/rpc/yarpc.go

```diff
@@ -306,6 +334,19 @@ func defaultYARPCDispatcher(_ service.Host, cfg yarpc.Config) (*yarpc.Dispatcher
 	return yarpc.NewDispatcher(cfg), nil
 }
 
+type yarpcStarterFn func(dispatcher *yarpc.Dispatcher) error
+
+// RegisterStarter allows you to override the YARPC dispatcher start, e.g. attach some metrics with start.
+func RegisterStarter(starterFN yarpcStarterFn) {
+	_dispatcherMu.Lock()
+	defer _dispatcherMu.Unlock()
+	_starterFn = starterFN
+}
+
+func defaultYARPCStarter(dispatcher *yarpc.Dispatcher) error {
```

Comment:
Do we even need _startedFn at all? Why can't we call c.dispatcher.Start() in line 152?